sites/fast-website/src/app/components/fast-frame/fast-frame.ts

```diff
@@ -96,7 +98,16 @@ export class FastFrame extends FASTElement {
     public borderRadius: number = 3;
     borderRadiusChanged() {
         DOM.queueUpdate(() => {
+            const maxCheckboxRadius = 8;
+
             controlCornerRadius.setValueFor(this.preview, this.borderRadius);
+
+            // We don't want checkboxes to look like radios, this is a temporary fix to a longstanding problem
```

Comment:
I think we can do this w/ just a CSS override which is certainly going to be more performant, https://stackblitz.com/edit/web-platform-lzq1d8?file=index.html is the basic approach and would use part selectors. Does that seem like it would work?

Reply:
I don't really have a preference in implementation - this portion of the homepage is going to go away soon, but Jason mentioned it so I put something together. I'll look to revisit this later.